scripts/pre_commit_linter.py

```diff
@@ -687,7 +693,11 @@ def _lint_all_files(
         Stylelint outputs.
     """
 
-    python_utils.PRINT('Starting Js, Ts, Python, HTML, and CSS linter...')
+    if CIRCLECI_ENV:
```

Comment:
Let's make it more generic to `IS_CI`. the thing is that we're moving the linting away from CircleCI entirely to Github Actions.

Reply:
Done.